elasticdl/pkg/main/main.go

```diff
@@ -34,7 +34,7 @@ func main() {
 	flag.Parse()
 	address := fmt.Sprintf("%s:%d", os.Getenv("MY_POD_IP"), *port)
 	serverDone := make(chan bool)
-	ps.NewServer(*psID, *optType, *optArgs).Run(address, serverDone)
+	ps.NewServer(*psID, *optType, *optArgs, *masterAddr, int32(*evaluationSteps)).Run(address, serverDone)
```

Comment:
A minor comment: don't use int32 but int.

Reply:
Changed to int.